planner/optimize.go

```diff
@@ -176,36 +175,36 @@ func handleInvalidBindRecord(ctx context.Context, sctx sessionctx.Context, stmtN
 		return
 	}
 	sessionHandle := sctx.Value(bindinfo.SessionBindInfoKeyType).(*bindinfo.SessionHandle)
-	bindMeta := sessionHandle.GetBindRecord(normdOrigSQL, sctx.GetSessionVars().CurrentDB)
-	if bindMeta != nil {
-		bindMeta.Status = bindinfo.Invalid
+	bindRecord := sessionHandle.GetBindRecord(normdOrigSQL, sctx.GetSessionVars().CurrentDB)
+	if bindRecord != nil {
+		bindRecord.FirstUsingBinding().Status = bindinfo.Invalid
```

Comment:
Can we make sure that the return value here must be not nil?

Reply:
Yes, because if it is nil, we cannot use it to build new plan and knows that it is invalid.